roles/tomcat/defaults/main.yml

```diff
@@ -5,7 +5,6 @@ selinux_enabled: false
 # mirsg.tomcat: java settings
 java_home: /usr/lib/jvm/jre
 java_profile_d: "/etc/profile.d"
```

Comment:
`java_home` and `java_profile_d` are still required for setting the Tomcat confirguration